contractsV5/KyberHintHandler.sol

```diff
@@ -223,85 +184,127 @@ contract KyberHintParser is IKyberHint, Utils {
     {
         bytes memory opcode = hint.slice(indexToContinueFrom, 1);
         bytes32 opcodeHash = keccak256(opcode);
-        ReservesHint memory reserves;
+        ReservesHint memory reservesHint;
 
         if (isTokenToEth) {
-            reserves = tradeHint.tokenToEthReserves;
+            reservesHint = tradeHint.tokenToEthReserves;
         } else {
-            reserves = tradeHint.ethToTokenReserves;
+            reservesHint = tradeHint.ethToTokenReserves;
         }
 
         indexToContinueFrom += 1;
+        reservesHint.invalidHint = false;
 
-        if (opcodeHash == keccak256(END_OPCODE)) {
+        if (opcodeHash == END_KECCAK) {
```

Comment:
seems to me more readable with consistent naming.

so can change opcodeHash to opcodeKeccak
or change END_KECCAK to END_HASH

Reply:
Changed to opcodeHash to opecodeKeccak for readability.